scripts/write-json.js

```diff
@@ -0,0 +1,13 @@
+const { promises: { readFile, writeFile } } = require('fs');
+const {convertToJson} = require("../scripts/utils")
+
+module.exports = async function writeJSON(readPath, writePAth){
+    try{
+        const readContent = await readFile(readPath,'utf-8');
+        const jsonContent = convertToJson(readContent);
+
+        await writeFile(writePAth,JSON.stringify(jsonContent));
+    }catch(error){
+        console.log(error)
```

Comment:
Instead of catching the error for all three operations, you can use nested try catch or different try catch scopes to better write tests that would allow you know which out of these three operations (`readFile`, `convertToJson`, `writeFile`) has failed.

Reply:
done